app/assets/javascripts/controllers/cloud_volume/cloud_volume_form_controller.js

```diff
@@ -14,7 +14,7 @@ ManageIQ.angular.app.controller('cloudVolumeFormController', ['$http', '$scope',
       { type: "io1", name: "Provisioned IOPS SSD (IO1)" },
       { type: "st1", name: "Throughput Optimized HDD (ST1)" },
       { type: "sc1", name: "Cold HDD (SC1)" },
-      { type: "magnetic", name: "Magnetic" },
+      { type: "standard", name: "Magnetic" },
```

Comment:
AWS does not provide an API for this; we initially thought about making `CloudVolumeType` model where storage managers would be able to list their options (for example in OpenStack admins are allowed to define different volume types so it would make sense).

My concern is that there are specific rules in the UI that need to be handled with regards to the chosen volume type, so I am not sure if providing it through API/backend would benefit much. Still, we could use something like this for [regions](https://github.com/ManageIQ/manageiq-providers-amazon/blob/master/app/models/manageiq/providers/amazon/regions.rb) or [instance types](https://github.com/ManageIQ/manageiq-providers-amazon/blob/master/app/models/manageiq/providers/amazon/instance_types.rb). Would that make any sense?